mindmeld/models/model.py

```diff
@@ -1286,8 +1287,9 @@ def decode(self, tags_by_example, **kwargs):
         """
         # TODO: support decoding multiple queries at once
         examples = kwargs["examples"]
+        sys_resolver = kwargs.get("sys_resolver") or DucklingRecognizer.get_instance()
```

Comment:
sys_resolver should be passed into the model config's constructor instead of hijacking the instance here.